ruby/Dockerfile

```diff
@@ -1,23 +1,29 @@
-FROM ruby:2.7.2
-ARG UNAME=app
+ARG RUBY_VERSION=2.7.2
+FROM ruby:${RUBY_VERSION}
+ARG BUNDLER_VERSION=2.1.4
```

Comment:
Parameterizing `RUBY_VERSION` and `BUNLDER_VERSION` seems like a good idea to me.